compiler/wasm/src/lib.rs

```diff
@@ -29,14 +31,21 @@ pub struct BuildInfo {
 }
 
 #[wasm_bindgen]
-pub fn init_log_level(level: String) {
+pub fn init_log_level(filter: String) {
```

Comment:
whats the difference between filter and level?

minor nit because the method is called init_log_level and its now taking in a filter variable

Reply:
Did not want to break it this way. Filter allows to limit events scope to module, file, etc., allows to set level per scope, while level would pick one of levels DEBUG/TRACE/etc.

Reply:
can you open up an issue for this and make a followup PR?